models/repos.js

```diff
@@ -144,6 +145,27 @@ function reposModel (state, bus) {
     document.body.appendChild(modal)
   })
 
+  bus.on('dats:edit-start', function (key) {
+    assert.equal(typeof key, 'string', 'dats:edit-start: key should be type string')
+    var dat = state.repos.values.find(function (dat) {
```

Comment:
I think we talked about always using the key as a buffer internally and only using the hex encoding for external apis. Shall we go with this here too?